client/rest-high-level/src/main/java/org/elasticsearch/client/watcher/PutWatchRequest.java

```diff
@@ -136,20 +109,20 @@ public void setVersion(long version) {
     }
 
     @Override
-    public ActionRequestValidationException validate() {
-        ActionRequestValidationException validationException = null;
+    public Optional<ValidationException> validate() {
```

Comment:
can we move this to the constructor a-la https://github.com/elastic/elasticsearch/blob/master/client/rest-high-level/src/main/java/org/elasticsearch/client/watcher/DeactivateWatchRequest.java#L29-L37